src/allmydata/test/cli/test_backup.py

```diff
@@ -86,7 +100,7 @@ def do_backup(verbose=False):
         d.addCallback(lambda res: do_backup(True))
         def _check0(args):
             (rc, out, err) = args
-            self.failUnlessReallyEqual(err, "")
+            self.assertEqual(len(err), 0)
```

Comment:
This makes the failure message somewhat worse.  Can you stick the content of `err` back into the message?